core/src/main/java/com/oath/oak/OldMemoryManager.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2018 Oath Inc.
+ * Licensed under the terms of the Apache 2.0 license.
+ * Please see LICENSE file in the project root for terms.
+ */
+
+package com.oath.oak;
+
+import com.oath.oak.NativeAllocator.OakNativeMemoryAllocator;
+import java.nio.ByteBuffer;
+
+
+public class OldMemoryManager {
```

Comment:
What does it mean "OldMemoryManager"? Name it anyhow different. Handle-based-Meory-Manager? It would be the best to use it for "Empty Memory Manager" as Eddie asked for...